src/overlays/actors/ovl_Obj_Kendo_Kanban/z_obj_kendo_kanban.c

```diff
@@ -1,28 +1,48 @@
 /*
  * File: z_obj_kendo_kanban.c
  * Overlay: ovl_Obj_Kendo_Kanban
- * Description: Swordsman's School - Cuttable Board
+ * Description: Swordsman's School - Cuttable Board at back of school
```

Comment:
I don't really think this clarification is necessary, what do others think?

Reply:
I added it since it was a cause of confusion for myself. Alongisde the practice log, the context is probably unnecessary, but if viewed in isolation, at least I confused it with the practice log since I wasn't aware of any other cuttable thing in the swordsman's school.

That said, figured it would have helped me for it to have been there, so it probably will help someone else at some point, but I won't argue for it past this comment. If you (or others) confirm opposition to this change, will undo.

Reply:
Personally I don't see a reason not to have, and if it is beneficial for some we can probably just keep it.